test/parallel/test-buffer-alloc.js

```diff
@@ -1305,7 +1304,7 @@ assert.throws(function() {
 
   // try to slice a zero length Buffer
   // see https://github.com/joyent/node/issues/5881
-  SlowBuffer(0).slice(0, 1);
+  Buffer.alloc(0).slice(0, 1);
```

Comment:
It makes no difference. alloc, allocUnsafe, and allocUnsafeSlow all do the same thing when the size is zero.